apps/cowswap-frontend/src/modules/trade/utils/getReceiveAmountInfo.ts

```diff
@@ -0,0 +1,95 @@
+import { isSellOrder } from '@cowprotocol/common-utils'
+import { type OrderParameters, getQuoteAmountsAndCosts, QuoteAmountsAndCosts } from '@cowprotocol/cow-sdk'
+import { Currency, CurrencyAmount, Percent, Price } from '@uniswap/sdk-core'
+
+import { DirectedReceiveAmounts, ReceiveAmountInfo } from '../types'
+
+export function getDirectedReceiveAmounts(info: ReceiveAmountInfo): DirectedReceiveAmounts {
```

Comment:
What does `directed receive amount` mean?

Reply:
Direction means sell or buy.
Directed means we return an amount depending on the order direction (sell/buy).
I'm happy to rename the function if you have better name :)

Reply:
Maybe `order type receive amount`?

Reply:
Thanks! Renamed in another PR:
https://github.com/cowprotocol/cowswap/pull/4502/commits/5743f6e1faa8bdacdc597064a37d6a440a96447d